src/app/routes/registration/enrolment-form/enrolment-form.component.spec.ts

```diff
@@ -93,7 +82,8 @@ describe('EnrolmentFormComponent', () => {
       expect(submit.nativeElement.disabled).toBeTruthy();
     });
 
-    it('should have disabled on-chain checkbox when registration types do not contains onChain type', () => {
+    // TODO: probably this test is no longer valid. Need to be checked.
+    xit('should have disabled on-chain checkbox when registration types do not contains onChain type', () => {
```

Comment:
I would check it and if it not used, just remove it. Otherwise you will forget it.

Reply:
Yeah, you're right! I will remove it.